nemo/collections/asr/losses/rnnt.py

```diff
@@ -173,6 +178,91 @@ def resolve_rnnt_loss(loss_name: str, blank_idx: int, loss_kwargs: dict = None)
     return loss_func
 
 
+class RNNTLossPytorch(Loss):
```

Comment:
Let's put this in another file rather than here which makes it confusing with the loss main class for RNNT

Reply:
Done.